packages/field-plugin/src/messaging/pluginMessage/containerToPluginMessage/index.ts

```diff
@@ -8,3 +8,4 @@ export * from './ContextRequestMessage'
 export * from './MessageToPlugin'
 export * from './StoryData'
 export * from './Asset'
+export * from './PromptAIResponseMessage'
```

Comment:
Can we move it closer to `MessageToPlugin` export? Just for the future, so we have ordered the export by type of content since StoryData and Asset are just data returned and not message-related.